data_generator/arangoPushTest.js

```diff
@@ -0,0 +1,50 @@
+const fs = require("fs");
+const Database = require("arangojs").Database;
+const aql = require("arangojs");
+const db = new Database("http://127.0.0.1:8529");
+db.useBasicAuth("root", "datasaver");
```

Comment:
make sure to put your username and pass to an env file and gitignore it

Reply:
Thanks, good idea